modules/fedimint-lnv2-server/src/db.rs

```diff
@@ -217,3 +226,47 @@ impl_db_lookup!(
     key = IncomingContractIndexKey,
     query_prefix = IncomingContractIndexPrefix
 );
+
+pub async fn migrate_to_v1(
+    mut ctx: ServerModuleDbMigrationFnContext<'_, Lightning>,
+) -> Result<(), anyhow::Error> {
+    let mut contracts = BTreeMap::new();
+    let mut index = 0;
+
+    let mut stream = ctx.get_typed_module_history_stream().await;
+
+    while let Some(item) = stream.next().await {
```

Comment:
Nit: could use `.zip(0..)` to avoid mutable `index`, which is even more confusing as the _name_ is reused later in this function.